diff --git a/target/i386/sev.c b/target/i386/sev.c
index 87c84ff1d4..3b32881cdc 100644
--- a/target/i386/sev.c
+++ b/target/i386/sev.c
@@ -1538,8 +1538,7 @@ snp_metadata_desc_to_page_type(int desc_type)
     case SEV_DESC_TYPE_SNP_SEC_MEM: return KVM_SEV_SNP_PAGE_TYPE_ZERO;
     case SEV_DESC_TYPE_SNP_SECRETS: return KVM_SEV_SNP_PAGE_TYPE_SECRETS;
     case SEV_DESC_TYPE_CPUID: return KVM_SEV_SNP_PAGE_TYPE_CPUID;
-    case SEV_DESC_TYPE_SNP_KERNEL_HASHES: return KVM_SEV_SNP_PAGE_TYPE_NORMAL;
-    default: return -1;
+    default: return KVM_SEV_SNP_PAGE_TYPE_ZERO;
     }
 }
 
